python/lsst/pipe/tasks/photoCal.py

```diff
@@ -36,6 +35,7 @@
 import lsst.pipe.base as pipeBase
 import lsst.afw.table as afwTable
 from lsst.afw.image import abMagFromFlux, abMagErrFromFluxErr, fluxFromABMag, Calib
+from lsst.afw.math import Random
```

Comment:
Import as `afwMath` so that it's clear where `Random` comes from when it's used 600+ lines below?